.github/workflows/on-push.yml

```diff
@@ -36,6 +36,14 @@ jobs:
         run: |
           echo "Validating changelog for PR: $PR_NUMBER with Name: $PR_TITLE"
           poetry run python ./Utils/github_workflow_scripts/changelog_validation_scripts/validate_changelog.py -n "$PR_NUMBER" -t "$PR_TITLE"
+      - name: Comment Changelog In PR Description
```

Comment:
Consider making this run only if the changelog validation passed (`needs: validate-changelog`) instead of `continue-on-error`

Reply:
wdym? it already runs **only** if validate was successful. (its like the behaivor you described, `needs: validate-changelog`)
I didn't see a reason to fail the whole job in case the comment didn't work.